templates/image-builder.yml

```diff
@@ -103,6 +103,18 @@ objects:
                 key: dsn
                 name: "${GLITCHTIP_DSN_NAME}"
                 optional: true
+          - name: RECOMMENDATIONS_URL
+            value: "${RECOMMENDATIONS_URL}"
+          - name: RECOMMENDATIONS_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                key: client_id
+                name: recommendations-secrets
+          - name: RECOMMENDATIONS_CLIENT_SECRET
+            valueFrom:
+              secretKeyRef:
+                key: client_secret
+                name: recommendations-secrets
```

Comment:
recommendations-secrets are currently not present. We can either mark them as optional, or provision the secrets in the namespaces first.

Reply:
add it as optional

Reply:
well you'd need to mark the client_id as optional as well.